### PR TITLE
Prefix caching with version number and do something weird to generate…

### PR DESCRIPTION
**Pull request by benjaoming**
… install_location_hash based on version instead of perceived location of user data, which was useless anyways #3722

### CODE REVIEW

File: kalite/distributed/settings.py [hunk 1]
```diff
@@ -160,17 +159,13 @@ def USER_FACING_PORT():
...
 #   EXCEPT: if CACHE_TIME=0
 if CACHE_TIME != 0:  # None can mean infinite caching to some functions
-    # We can bump the version and forget about version info
-    version_info = version.VERSION_INFO()
-    if version.VERSION in version_info:
-        KEY_PREFIX = version_info[version.VERSION]["git_commit"][0:6]  # new cache for every build
-    else:
-        KEY_PREFIX = "unknown_version"
+    # When we change versions, cache changes, too
+    KEY_PREFIX = ".".join(version.VERSION)
```
Comment (MCGallaspy) [inline kalite/distributed/settings.py]: For anyone wondering like me, here's what [KEY_PREFIX](https://docs.djangoproject.com/en/1.5/ref/settings/#key-prefix) does. [See also](https://docs.djangoproject.com/en/1.5/topics/cache/#cache-key-prefixing).

**Comment by MCGallaspy** [comment]
Your test failures here should now be resolved in `develop`.

**Comment by MCGallaspy** [comment]
Otherwise looks ok, just merge `develop` back in.

**Comment by aronasorman** [comment]
I'm encountering a similar problem due to VERSION_INFO getting loaded. Will merge this now to get it into the central server.

**Pull request merged**